FAQ.md

```diff
@@ -203,3 +203,71 @@ targets:
 ```
 
 Also, you can also check out the example configuration in the Mockito repository. 
+
+
+## How do I mock a `sealed` class?
+
+Suppose that you have code such as:
+
+```dart
+class Bar {
+  int value;
+
+  Bar(this.value);
+
+  Bar.zero() : value = 0;
+}
+
+class Foo {
+  Bar someMethod(int value) => Bar(value);
+}
+```
+
+and now you want to mock `Foo`. The generated implementation for `MockFoo`
```

Comment:
I think this is a bit of a leap from wanting to mock `Foo` to having a generated implementation. Maybe something like, "A mock implementation of `Foo`, generated by `@GenerateNiceMocks([MockSpec<Foo>()])`, needs to ..."

Reply:
I do like that better.  Fixed.